---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* B.S. in Chemistry and Biophysics, Macalester College, 2015
* M.S. in Bioinformatics, ETH Zürich, 2020

Work experience
======
* 2020: Master Thesis in Protein Language Modeling
  * IBM Research, Cognitive Computing and Industry Solutions
  * Built unsupervised NLP language models (embeddings) for protein sequences
  * Applied NER, neural machine translation, sentiment analysis to protein sequence domain
  * Used embeddings for token-level classification tasks such as protein structure prediction, and sentence-level tasks like protein function prediction
  * Tools used: Pyhton, PyTorch, Git, Huggingface libraries

* Fall 2019: Computer Vision and Natural Language Understanding Intern
  * Novartis Institutes for BioMedical Research (NIBR)
  * Implemented a multimodal learning model to automatically generate text reports from biomedical images (chest X-rays)
  * Adapted established NLP and computer vision models to biomedical domain
  * Wrote a technical report and presented the findings to other Data Science Analytics team
  * Tools used: Python, PyTorch, JupyterHUB, tmux, Git

* 2019: Programming research assistant
  * Decision Science Laboratory (DeSciL), ETH Zürich
  * Developed Django web apps for Behavioral Economics research experiments
  * Participated in meetings with Economics researchers to adjust project design
  * Tools used: Python, Django, JavaScript, CSS, HTML, OTree

* 2016-2017: Bioinformatician
  * Specifica Inc. in Basel and Santa Fe, NM
  * Implemented genomic data query, processing and analytics workflows in R and Unix shell
  * Established sequencing protocol of yeast-phage display selections against disease biomarkers
  * Identified antibody drug candidates with gene annotation and clonotyping algorithms
  * Tools: R, Python, Bash shell, various Bioinformatics libraries

* 2015-2016: Research Assistant at Viral Biophysics Laboratory
  * University of Washington, Seattle
  * Introduced potential HIV-1 envelope-stabilizing mutations with RosettaDesign and MD simulations
  * Contributed to the development of a new antibody-independent HIV Env purification method
  * Modeled HIV-1 Env biophysical stability and dynamics from light scattering, Hydrogen-Deuterium Exchange Mass Spectroscopy and Small-angle X-ray scattering experiments


Skills
======
* Programming
  * Python
  * R
  * Java (basics)

* Machine Learning
  * PyTorch
  * Scikit-learn
  * HuggingFace libraries
  * AllenNLP
  * Gensim
  * NumPy

* Big data
  * Hadoop framework 
  * MySQL 
  * HPC and LSF cluster
  * JupyterHub 
  * Pandas

* DevOps
  * scripting in Bash and Python
  * Docker config management
  * Git version control
  * Jenkins continuous integration
  * Azure cloud

Publications
======
  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
<!-- Talks
======
  <ul>{% for post in site.talks %}
    {% include archive-single-talk-cv.html %}
  {% endfor %}</ul>
  
   -->

Service and leadership
======
* Member at Bioinformatics Student Committee (CBB) at ETH Zürich
  * Organized career events for Bioinformatics students, attended VIS meetings, 
  * 2018-2020
  
* Kullu Volunteering Project 
  * http://volunteers.uwcrcn.no/kullu-project-northern-india-2 
  * Instructed First Aid in local schools in Himachal Pradesh
  * August 2010 - May 2011

Awards
======
* University of Washington Top Scholar Award (2015)
* HHMI Young Researchers Scholarship (2013)
* Shelby Davis United World College Scholarship (2010)